syntax = "proto3";

package notes.sync;

service SyncService {
  rpc AddNote (NoteResponse) returns (EmptyResponse);
  rpc UpdateNote (NoteResponse) returns (EmptyResponse);
  rpc DeleteNote (DeleteResponse) returns (EmptyResponse);

}

message NoteResponse {
  int64 note_id = 1;  
  string full_text = 2;      
}

message DeleteResponse {
  int64 note_id = 1;  
}

message EmptyResponse {}
